framework/src/modules/chain/blocks/blocks.js

```diff
@@ -493,6 +493,24 @@ class Blocks extends EventEmitter {
 		}
 	}
 
+	/**
+	 * Returns the largest common block between ids.ids and the database blocks table
```

Comment:
Typo? Prefer IDs (also for the variable naming in the `getHighestCommonBlock` function.